code/modules/projectiles/projectile/beams.dm

```diff
@@ -222,4 +222,33 @@
 /obj/item/projectile/beam/stun/med
 	name = "stun beam"
 	icon_state = "stun"
-	agony = 30
\ No newline at end of file
+	agony = 30
+
+/obj/item/projectile/beam/medbeam
+	name = "healing beam"
+	icon_state = "healbeam"
+	damage = 0

```

Comment:
Did you try using the `nodamage` var instead of doing it this way?

Reply:
Damage was set to 0 instead of using nodamage to keep it consistent with weapons like lasertag guns. It now uses nodamage var.